nemo_text_processing/inverse_text_normalization/ar/taggers/decimal.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2021, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
```

Comment:
could you update the top line of all headers to 
Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.